experimental/ir/lower_validate.go

```diff
@@ -38,6 +38,10 @@ import (
 	"github.com/bufbuild/protocompile/internal/ext/slicesx"
 )
 
+const (
+	TagUnusedImport = "unused-import"
```

Comment:
Two things: I would like to have a single package, maybe `report/tags`, to hold these constants.

I would like for the tags to be namespaced, e.g. `protobuf:unused_import`. I also want them to be normal Protobuf identifiers.

Reply:
Does https://github.com/bufbuild/protocompile/pull/620/commits/4453d6a1034c58ff06bd308410dae3629c6d4e3a look like what you're thinking?